Cargo.toml

```diff
@@ -183,9 +183,7 @@ clap_complete = "4.3"
 colored = "2.0.0"
 colored_json = "3.2.0"
 console = "0.15.7"
-# Controller PR revision until merged.
-# https://github.com/cartridge-gg/controller/pull/1454
-account_sdk = { git = "https://github.com/cartridge-gg/controller", rev = "dbbe0353d64de743739d425f8aab91ca3ac0e16f" }
+account_sdk = { git = "https://github.com/cartridge-gg/controller" }
```

Comment:
We should use at least a tag or a rev. Since without any of them we rely on the cargo lock only.

Or is it the format required to work with `dojo.c`?